{
  "cveId": "CVE-2018-25111",
  "eventName": "Initial Analysis",
  "cveChangeId": "77512958-3BB3-44F9-A518-D658EB76EE1D",
  "sourceIdentifier": "nvd@nist.gov",
  "created": "2025-06-16T16:25:41.307",
  "details": [
    {
      "action": "Added",
      "type": "CVSS V3.1",
      "newValue": "AV:L/AC:L/PR:L/UI:N/S:U/C:L/I:L/A:N"
    },
    {
      "action": "Added",
      "type": "CPE Configuration",
      "newValue": "OR\n          *cpe:2.3:a:django-helpdesk_project:django-helpdesk:*:*:*:*:*:*:*:* versions up to (excluding) 1.0.0"
    },
    {
      "action": "Added",
      "type": "Reference Type",
      "newValue": "MITRE: https://github.com/django-helpdesk/django-helpdesk/issues/591 Types: Issue Tracking"
    },
    {
      "action": "Added",
      "type": "Reference Type",
      "newValue": "CISA-ADP: https://github.com/django-helpdesk/django-helpdesk/pull/1120 Types: Exploit, Patch, Third Party Advisory"
    },
    {
      "action": "Added",
      "type": "Reference Type",
      "newValue": "MITRE: https://github.com/django-helpdesk/django-helpdesk/pull/1120 Types: Exploit, Patch, Third Party Advisory"
    },
    {
      "action": "Added",
      "type": "Reference Type",
      "newValue": "MITRE: https://github.com/django-helpdesk/django-helpdesk/releases/tag/v1.0.0 Types: Release Notes"
    }
  ]
}